src/pages/dataTableUtilities/reducer/tableReducers.js

```diff
@@ -127,6 +127,15 @@ export const refreshData = state => {
   return { ...state, data: newData, searchTerm: '', showAll: true };
 };
 
+/**
+ * Override for refreshData for cash register. Filtering and sorting is deferred to state-to-prop
+ * selectors.
+ */
+export const refreshCashRegister = state => {
```

Comment:
Since there is already the action which closes the modal in the add action, you could batch dispatch both the `refresh` and the `closeModal` actions, which I think is cleaner and ensures actions are doing their action rather than other random things?

Reply:
3c3541c126088ba8d548246dc360baf775d57d2e. Omitted batching as it seems to prevent the cash register selectors from recomputing the page data (would expect the state to get updated by both reducers before triggering `mapStateToProps` before the single render cycle, but this doesn't seem to be the case).

Reply:
weird .. !